README.md

```diff
@@ -120,7 +124,16 @@ bytestring_to_pdf(b"<html><h1>Hello Chrome!</h1></html>", open('myfile.pdf'), **
       def get_filename(self):
           return "my_file_{}.pdf".format(now().strftime('Y-m-d'))
   ```
-  
+- How do I render context using the CBV mixin?
+
+  Add your context to the `template_context` object, this should be a `dict`:
+  ```python
+  class MyView(PDFViewMixin, TemplateView):
+      def get(self, request, *args, **kwargs):
+          data = request.GET.get('example')
+          self.template_context['example_data'] = data
+          ...
+  ```
```

Comment:
I don't think this is a good idea, why not simply override `TemplateView.get_context_data` here ?

```python
class MyView(PDFViewMixin, TemplateView):
    def get_context_data(self, **kwargs):
        context = super(MyView, self).get_context_data(**kwargs)
        context['example_data'] = self.request.GET.get('example')
        return context
```

Reply:
Agreed--the idea is to mixin the PDFViewMixin with an existing CBV.  Encouraging users to override `get` will lead people breaking the functionality of the core CBVs.

Reply:
I've done this quite often and thought it would be helpful in the base class. However, with your goal in mind, it definitely makes sense to keep them separate. I'll rectify the PR.

I think it may be useful to have a recipe in the documentation to show how to provide context to a given template.